draft-ietf-tls-tls13.md

```diff
@@ -2069,8 +2069,8 @@ When this message will be sent:
 > Servers send this message in response to a ClientHello
 message when it was able to find an acceptable set of algorithms and
 groups that are mutually supported, but
-the client's ClientKeyShare did not contain an acceptable
-offer. If it cannot find such a match, it will respond with a
+the client's ClientKeyShareOffers did not contain an acceptable
```

Comment:
I don't think this is an improvement. We generally refer to the extension, not to the
inner structs.